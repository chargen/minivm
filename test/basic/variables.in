aaa = 3 * 4 / 5 + 6.7 * 7
aab = aaa * 3.1 + aaa / 7.2
print aaa + aab
