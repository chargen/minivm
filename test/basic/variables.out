207.281666667
